airbyte-integrations/connectors/source-chartmogul/source_chartmogul/source.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC
+from base64 import b64encode
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.auth import TokenAuthenticator
+
+
+# Basic full refresh stream
+class ChartmogulStream(HttpStream, ABC):
+    url_base = "https://api.chartmogul.com"
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        yield from response.json().get("entries", [])
+
+
+class Customers(ChartmogulStream):
+    primary_key = "uuid"
```

Comment:
Customer schema has an `id` and an `uuid` field, what makes `uuid` more legit than `id` for a primary key?

Reply:
good question. Both seem to be unique according to documentation, but i picked uuid since, well, it's uuid and should always be unique.. Not the best argument haha 

Is there any performance impact on string vs int primary key?

Reply:
int is probably better for performance in for a PK (easier to sort and lighter)